relayer/relays/execution/main.go

```diff
@@ -97,6 +97,13 @@ func (r *Relay) Start(ctx context.Context, eg *errgroup.Group) error {
 				return err
 			}
 
+			if executionHeaderState.BlockNumber == 0 {
+				log.WithFields(log.Fields{
+					"channelId": r.config.Source.ChannelID,
+				}).Info("Beacon execution state syncing not started,wait...")
+				continue
+			}
+
```

Comment:
This is great, thanks for adding! On Goerli it takes quite a while and this message is much more clear than `no contract found at address`. 🙃